lib/src/bbox.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:turf/helpers.dart';
+import 'package:turf/meta.dart';
+
+BBox bbox(GeoJSONObject geoJson, {Map<String, dynamic>? options}) {
+  if (geoJson.bbox != null && options != null && true != options['recompute']) {
+    return geoJson.bbox!;
+  }
+
+  var result = [double.infinity, double.infinity, double.negativeInfinity, double.negativeInfinity];
```

Comment:
My idea would be to put the values already here  into a BBox `BBox(lng1: double.infinity, .....)`, so we can use lat/lng instead of indexes inside the coordEach callback, which gives more clarity imo.